### PR TITLE
update(find-test-names): update to v1.24

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- new `requiredTags` property on results
    https://github.com/bahmutov/find-test-names/pull/52
    https://github.com/bahmutov/find-test-names/releases/tag/v1.24.0

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

**Review by Michsior14 — APPROVED** [review]

File: types/find-test-names/index.d.ts [hunk 1]
```diff
@@ -32,6 +33,7 @@ export interface TestInfo {
 }
 export interface Suite {
     name: string;
+    requiredTags: string[] | undefined;
```
Comment (Michsior14) [inline types/find-test-names/index.d.ts:36]: Shouldn't this be optional? Same in Test.
https://github.com/bahmutov/find-test-names#requiredtags

Comment (peterblazejewicz) [reply types/find-test-names/index.d.ts:36]: yes, the outcome would be same though, let me amend

Comment (peterblazejewicz) [reply types/find-test-names/index.d.ts]: amended

Comment (Michsior14) [reply types/find-test-names/index.d.ts]: > yes, the outcome would be same though, let me amend

Not exactly, this way you don't need to specify it  in options at all ;)

**Review by Michsior14 — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge

**Pull request merged**